### PR TITLE
Improved wording of some OT questions and answers

### PR DESCRIPTION
**Pull request by tombogle**
This was especially to eliminate archaic KJV language

- - -
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/130)

### CODE REVIEW

File: Transcelerator/TxlQuestions.xml [hunk 1]
```diff
@@ -96354,32 +96357,52 @@
         </Questions>
         <Questions scrref="LEV 25.35-43" startref="3025035" endref="3025043" exclude="false" user="false">
           <Q>How was an Israelite to treat a poor Israelite whom he had purchased?</Q>
-          <A>Do not charge him usury nor rule over him with rigor. Treat him as a hired servant rather than a bond servant. Also release him during the year of jubilee. (35-43)</A>
+          <A>He was not to charge him usury/interest. (36-37)</A>
+          <A>He was not to treat him (harshly) as a slave. (39, 42-43)</A>
+          <A>Instead, he was treat him as a hired servant. (40)</A>
```
Comment (andrew-polk) [inline Transcelerator/TxlQuestions.xml]: Should be, "he was to treat..."

**Review by andrew-polk — CHANGES_REQUESTED** [review]

**Review by andrew-polk — COMMENTED** [review]
> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/130)* status: all files reviewed, 2 unresolved discussions (waiting on @tombogle)

**Comment by tombogle** [comment]
<details><summary><i>Previously, andrew-polk wrote…</i></summary><blockquote>

*(Reviewable was [unable](https://github.com/Reviewable/Reviewable/issues/14) to map [this GitHub inline comment thread](https://github.com/sillsdev/Transcelerator/pull/130#discussion_r1604017236) to the right spot — sorry!)*

Should be, "he was to treat..."
</blockquote></details>

I'm so glad you're here.

**Review by andrew-polk — COMMENTED** [review]
It's been a struggle to get that large file to work well in the various review/comparison tools. I thought I had added a comment about a typo for angel (angwl), but I don't see it.

**Review by andrew-polk — APPROVED** [review]

**Review by andrew-polk — COMMENTED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/Transcelerator/130)* status: all files reviewed, 1 unresolved discussion (waiting on @tombogle)

**Pull request merged**